stdlib-install.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/sh
+
+set -eu
+
+throwError() {
+  echo "\033[91m✗\033[0m $1." >&2
+  exit 1
+}
+
+logHappy() {
+  echo "\033[32m✔\033[0m $1"
+}
+
+# Pick the Agda executable to analyse
+# unless the caller has specified one
+if [ -z ${AGDA_EXEC-} ]; then
+    read -p "What's the name of your Agda executable (default: agda)? " AGDA_EXEC
+    if [ -z "$AGDA_EXEC" ]; then
+        AGDA_EXEC=agda
+    fi
+fi
+
+# Double check that the command exists
+if ! [ -x "$(command -v $AGDA_EXEC)" ]; then
+  throwError "'$AGDA_EXEC' is not a valid executable"
```

Comment:
Should we check if e.g. `$HOME/.cabal/bin/$AGDA_EXEC` exists and suggest that the user should be adding `$HOME/.cabal/bin/` to their `PATH`?

Reply:
You can get `cabal`'s default install path from `cabal` with `cabal path --installdir`